Sources/AppcuesKit/Presentation/UI/Components/AppcuesOptionSelect.swift

```diff
@@ -20,7 +20,16 @@ internal struct AppcuesOptionSelect: View {
         let errorTintColor = stepState.shouldShowError(for: model.id) ? Color(dynamicColor: model.errorLabel?.style?.foregroundColor) : nil
 
         VStack(alignment: style.horizontalAlignment, spacing: 0) {
-            TintedTextView(model: model.label, tintColor: errorTintColor)
+            HStack {
+                // nil-coalesce to .leading so a non-specified value defaults to leading-aligned
+                if HorizontalAlignment(string: model.label.style?.horizontalAlignment) ?? .leading != .leading {
+                    Spacer()
+                }
+                TintedTextView(model: model.label, tintColor: errorTintColor)
+                if HorizontalAlignment(string: model.label.style?.horizontalAlignment) != .trailing {
+                    Spacer()
+                }
+            }
```

Comment:
only other approach that comes to mind, and what I'm considering on the Android side - using nested `VStack` set to fill parent width, and then enforcing their own horizontal alignment within. This seems fine too.

Reply:
A nested `VStack` would be nice, but actually doesn't work (at least simply) because frame on the stack doesn't affect the layout within:
<img width="1522" alt="Screenshot 2023-01-03 at 1 35 58 PM" src="https://user-images.githubusercontent.com/845681/210420014-b655702e-eee4-4fb6-b742-67f444f73424.png">

Reply:
oh, darn, that seems kinda weird. On Compose it does fill the parent width, and then align the nested contents within that full width.